app/components/FilterMenu.tsx

```diff
@@ -56,4 +57,5 @@ export const FilterMenu = ({
 export interface FilterMenuProps {
   onClose?: () => void;
   children: ReactNode;
+  toNav?: () => void;
```

Comment:
Maybe there's a more generic name for this prop? Like `handleSubmit` or `onSubmit`? The component doesn't necessary "care" that we're using it to trigger navigation.